ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/CommandTransactionChecks.scala

```diff
@@ -423,6 +426,98 @@ abstract class CommandTransactionChecks
         }
       }
 
+      "permit fetching a divulged contract" in forAllMatchingFixtures {
+        case TestFixture(SandboxInMemory, ctx) =>
+        def pf(label: String, party: String) =
+          RecordField(label, Some(Value(Value.Sum.Party(party))))
+        val odArgs = Seq(pf("owner", owner), pf("delegate", delegate))
+        val delegatedCreate = simpleCreate(
+          ctx,
+          cid("SDVl3"),
```

Comment:
`cid` is a bad name... Can be Contract ID or Command ID, but I guess it is Command ID in this case:
```
private def cid(commandId: String) = s"$commandId"
```

Reply:
This function precedes this PR; do you want me to rename it?